pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -2153,15 +2135,13 @@ func handleReboot(getconfigCtx *getconfigContext) {
 	execReboot(state)
 }
 
-// Used by doBaseOsDeviceReboot only
-func startExecReboot() {
-
-	log.Infof("startExecReboot: scheduling exec reboot\n")
-
-	//timer was started, stop now
+// Used by doDeviceReboot only
+func startExecReboot(reasonStr string) {
+	//timer has already been started, return
 	if rebootTimer != nil {
-		rebootTimer.Stop()
+		return
```

Comment:
Why this change?

Reply:
No point of resetting this for device state machine induced reboot events. They are bound to get executed. Resetting rebootTimer makes sense for opCmd based reboot commands.